gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-bin.zip
```

Comment:
Is there something that requires the Gradle version downgrade?

Reply:
I think I re-ran `gradle wrapper` or something. Certainly I don't remember modifying this file by hand. Will revert.